doc/tutorials/enable_s5.rst

```diff
@@ -0,0 +1,364 @@
+
+   #. Use below commands to enable ``life_mngr.service`` and restart User VM.
+
+      .. code-block:: none
+
+         # chmod +x /usr/bin/life_mngr
+         # chmod +x /lib/systemd/system/life_mngr.service
+         # systemctl enable life_mngr.service
+         # reboot
+
+#. For WaaG VM, a reference daemon is offered to run:
+
+   a) Build ``life_mngr_win.exe`` application:
+
+      I) Go to ``acrn-hypervisor/misc/life_mngr``, then run
+         ``x86_64-w64-mingw32-gcc -g life_mngr_win.c -o life_mngr_win.exe`` under docker environment.
+
+      #) Or you can also enter ``acrn-hypervisor/misc`` directory and run ``make life_mngr``
+         to build ``life_mngr_win.exe``.
+
+      #) If there is no ``x86_64-w64-mingw32-gcc`` compiler, you need run ``swupd bundle-add c-basic-mingw``
+         to install it.
+
+   #) Set up a Windows environment:
+
+      I) Download the :kbd:`Visual Studio 2019` tool from `<https://visualstudio.microsoft.com/downloads/>`_,
+         and choose the two options in the below screenshots to install "Microsoft Visual C++ Redistributable
+         for Visual Studio 2015, 2017 and 2019 (x86 or X64)" in WaaG:
+
+         .. figure:: images/Microsoft-Visual-C-install-option-1.png
+
+         .. figure:: images/Microsoft-Visual-C-install-option-2.png
+
+      #) In WaaG, use the :kbd:`WIN + R` shortcut key, input "shell:startup", click :kbd:`OK`
+         and then copy ``life_mngr_win.exe`` application into this directory.
+
+         .. figure:: images/run-shell-startup.png
+
+         .. figure:: images/launch-startup.png
+
+   #) Restart the WaaG VM. The COM2 window will automatically open after reboots.
+
+         .. figure:: images/open-com-success.png
+
+#. If the Service VM is being shut down (transitioning to the S5 state), it can call
+   ``acrnctl stop vm-name`` to shut down the User VMs.  A script is offered as reference:
+
+   .. code-block:: bash
+
+      #! /bin/bash
+
+      MAX_ARRAY=6
+      
+      status="started"
+      rtvm="hard_rtvm"
+      
+      #set split code
+      OLD_IFS=$IFS
+      IFS=$' 	\n'
+      
+      vm_list=$(acrnctl list)
+      echo $vm_list
+      
+      array=($vm_list)
+      
+      num=${#array[@]}
+      echo "vm array size:" $num
+      
+      if [[ $num -gt $MAX_ARRAY ]]
+      then
+      	echo "not normal for acrnctl, or no VMs!"
+      	num=0
+      fi
+      
+      #to shutdown vms
+      flag=0
+      for ((i=0; i<$num; i+=2))  
+      do 
+      #	echo ${array[$i]} + ${array[$i+1]}
+      
+      	if [[ ${array[$i+1]} == $status ]]
+      	then
+      		echo "to shutdown: " ${array[$i]}
+      		acrnctl stop ${array[$i]}
+      		sleep 5s
+      		flag=1
+      	fi
+      done
+      
+      #check vm status for some time
+      check_times=4
+      while [[ $flag == 1 ]]
+      do
+      
+      	vm_list=$(acrnctl list)
+      	array=($vm_list)
+      	array=($vm_list)
+      
+      	num=${#array[@]}
+      	echo "vm status: " $vm_list
```

Comment:
What are you displaying here? Can you make the message clearer?

Reply:
here changed to: "living vms:"